src/utils/chunkArray.ts

```diff
@@ -0,0 +1,10 @@
+export function chunkArray(array: any[], chunkSize: number): any[][] {
```

Comment:
Would prefer to avoid any

Reply:
Yes. I mean, `unknown` does not add much value but was my first thought. Would you rather go with Generics?

Reply:
It added and handles types now!